<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="ejecutar" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>500</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="directorioEntrada" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>40</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="abrirEntrada" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>40</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxSize" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileMarca" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>150</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="abrirMarca" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>150</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePie" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>290</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="abrirPie" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>290</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="opacidadMarca" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>210</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectorioEntrada" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Directorio</string>
    </property>
    <property name="buddy" >
     <cstring>directorioEntrada</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labelSeccionFotos" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Fotos</string>
    </property>
   </widget>
   <widget class="Line" name="line1" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>20</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelMaxSize" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Tamaño máx.</string>
    </property>
   </widget>
   <widget class="Line" name="line2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>130</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFileMarca" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Imágen</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOpacidadMarca" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Opacidad</string>
    </property>
   </widget>
   <widget class="Line" name="line3" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>270</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilePie" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Imágen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="probar" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>500</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Prueba</string>
    </property>
   </widget>
   <widget class="QSlider" name="opacidadMarcaBar" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>210</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>100</number>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkMarca" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Marca de agua</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPie" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>250</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Firma 1</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboVMarca" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>180</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Arriba</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Medio</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Abajo</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboHMarca" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>180</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Izquierda</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Centro</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Derecha</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="comboVMarcaLabel" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>180</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación V.</string>
    </property>
   </widget>
   <widget class="QLabel" name="comboHMarcaLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>180</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación H.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboVPie" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>320</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Arriba</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Medio</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Abajo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="comboHPieLabel" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>320</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación H.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboHPie" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>320</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Izquierda</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Centro</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Derecha</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="comboVPieLabel" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>320</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación V.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboVPie2" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>440</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Arriba</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Medio</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Abajo</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="filePie2" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>410</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line3_2" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>390</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="abrirPie2" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>410</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>...</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPie2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>370</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text" >
     <string>Firma 2</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboHPie2" >
    <property name="enabled" >
     <bool>false</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>440</y>
      <width>88</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Izquierda</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Centro</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Derecha</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="comboVPie2Label" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>440</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación V.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilePie2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>410</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Imágen</string>
    </property>
   </widget>
   <widget class="QLabel" name="comboHPie2Label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>440</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Alineación H.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
